tests/cpp/tree/test_prediction_cache.h

```diff
@@ -0,0 +1,92 @@
+#include <gtest/gtest.h>
```

Comment:
Please include the license header, along with C header guard.

Reply:
done